app/js/beacons/beaconService.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * beaconService
+ * Management of Lighthouse beacons
+ */
+function beaconService($http, actions, flux, configService, alertService) {
+    'use strict';
+
+    function createBeacon(beacon) {
+        var request = [configService.api.base, 'beacons/', 'create'].join('');
+        $http.post(request, beacon).then(
+            // success
+            function (response) {
+                flux.dispatch(actions.createBeacon, {'response': response.data});
```

Comment:
I'm thinking we could stick a call to `alertService` in here after `flux.dispatch...` to note a successful creation. I'd like to do it in the `beaconModel`, but I think flux will complain about dispatching another action while one is in progress.